x-pack/legacy/plugins/apm/common/elasticsearch_fieldnames.ts

```diff
@@ -3,6 +3,9 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
+export const TIMESTAMP = 'timestamp';
```

Comment:
Where is this used (apart from the ES query)?

Reply:
It will be only used to return the `timestamp` property on ES query.

Reply:
And where do we use that?

Reply:
Removing it as I'm going to return the whole Transaction.

Reply:
Sounds good - but just to be clear: are you using timestamp anywhere?

Reply:
... because pretty sure it's either `@timestamp` or `timestamp.us`. But if we don't use it anywhere this doesn't matter.